### DISCUSSION TITLE
2023/07/16/my-site-starter/

### DISCUSSION THREAD

**Discussion by giscus** [longerking/longerking-blogs#3] [General]
# 2023/07/16/my-site-starter/

Just Work for Life(WL),  Be Your Longer King!

https://longerking.github.io/longerking-blogs/2023/07/16/my-site-starter/

**Comment by longerking** [comment]
建议测试gicus评论的有效性，目前存在的问题，博客无法获取到gicus写入的评论。

**Reply by longerking** [reply]
目前切换为：category: General。已经可以成功评论以及获取评论，赞一个！Fix a bug. :bug:

**Comment by longerking** [comment]
建议TODO

1. 如何使用HEXO一步一步搭建博客。
2. 介绍如何使用的 giscus 评论插件。
3. 使用Keep源码方式替代npm包安装keep-theme，方便做自定义修改。
